### DISCUSSION TITLE
Showcase request: Fair Internet Report's broadband speed tests map

### DISCUSSION THREAD

**Discussion by tbbuck** [visgl/deck.gl#7993] [Show and tell]
Hi there!

We used deck.gl to create a fun animated interactive map that shows the ~3.6 million internet speed tests that occurred on a single day in June 2023:

https://fairinternetreport.com/research/internet-speeds-map-single-day

I think it's a great illustration of what's possible with deck.gl, making use of incremental data updates, storing data in binary arrays and accessing them via offsets to try and keep things performant. We also use using custom shaders for fading the arcs in and out, and animating a white indicator across each one.

**Comment by Pessimistress** [comment]
Can you submit a PR to https://github.com/visgl/deck.gl/blob/master/website/showcase.json?

**Reply by tbbuck** [reply]
Thanks, happy to :-)

Please can you take a look at https://github.com/visgl/deck.gl/pull/8084 and let me know if we've missed anything?

**Comment by ashkanrdn** [comment]
> Hi there!
> 
> We used deck.gl to create a fun animated interactive map that shows the ~3.6 million internet speed tests that occurred on a single day in June 2023:
> 
> https://fairinternetreport.com/research/internet-speeds-map-single-day
> 
> I think it's a great illustration of what's possible with deck.gl, making use of incremental data updates, storing data in binary arrays and accessing them via offsets to try and keep things performant. We also use using custom shaders for fading the arcs in and out, and animating a white indicator across each one.

Hey, its such a cool visualization. is there any write up anywhere on the implementation? I'm trying to achieve something similar haven't had much luck.
